tests/foreman/cli/test_hammer.py

```diff
@@ -89,7 +89,7 @@ def traverse_command_tree():
 
     """
     differences = {}
-    raw_output = ssh.command('hammer full-help', output_format='plain').stdout
+    raw_output = ssh.command('hammer full-help').stdout
```

Comment:
This function is only called once, within one test in this module. Consider just moving it into the test, or passing `default_sat` from the test to remove one more use of `ssh.command`.

Reply:
moved into the test and removed the ssh import!